.github/workflows/rspec_rubocop.yml

```diff
@@ -32,10 +32,10 @@ jobs:
           bundler-version: latest
         - ruby-version: 2.7
           bundler-version: latest
-        # - ruby-version: 3.0
-        #   bundler-version: latest
-        # - ruby-version: 3.1
-        #   bundler-version: latest
+        - ruby-version: '3.0'
```

Comment:
Had to put it as float given that GitHub actions parses float `3.0` as integer `3` which causes the CI to install Ruby 3.2 instead.

Reply:
lol good catch! I just applied this in https://github.com/airbnb/ruby/pull/197